prototype2/common/CMakeLists.txt

```diff
@@ -73,6 +75,14 @@ set(DataSaveTest_INC
     TimeString.h)
 create_test_executable(DataSaveTest)
 
+set(DumpFileTest_SRC
+  DumpFileTest.cpp
+  TimeString.cpp)
+set(DumpFileTest_INC
+  DumpFile.h
+  TimeString.h)
```

Comment:
So I would suggest a more specific name such as DumpHDF5File or HDF5Writer or similar.

Reply:
I have thought about this and I think I would keep it as it. 1) No need to specify the format in the class name, especially after we phase out CSV. 2) It is not only a writer, but also a reader.